fixtures/_fixtures/logger.py

```diff
@@ -61,6 +62,13 @@ def _setUp(self):
             self.addCleanup(logger.removeHandler, self.handler)
 
 
+class StreamHandlerRaiseException(StreamHandler):
+    """Handler class that will raise an exception on formatting errors."""
+    def handleError(self, record):
+        ei = sys.exc_info()
+        raise ei[0], ei[1], ei[2]
```

Comment:
This will fail on 3.x - 
https://travis-ci.org/testing-cabal/fixtures/jobs/84212114#L213

six.reraise will be helpful (we already use six)